core/src/test/java/com/linecorp/armeria/client/HttpClientFactoryTest.java

```diff
@@ -103,4 +114,35 @@ protected HttpResponse doGet(ServiceRequestContext ctx,
             });
         }
     }
+
+    @Test
+    void useOpenSsl() throws ExecutionException, InterruptedException {
+        try (ClientFactory sslEnabledClient = ClientFactory.builder().tlsNoVerify().build();
+             ClientFactory sslDisabledClient = ClientFactory.builder().useOpenSsl(false).build();
```

Comment:
Since this option is valid only if ssl is enabled

```suggestion
        try (ClientFactory openSslEnabledClientFactory = ClientFactory.builder().tlsNoVerify().build();
             ClientFactory openSslDisabledClientFactory = ClientFactory.builder().useOpenSsl(false).build();
```